zh_cn/news/_posts/2025-04-14-ruby-3-4-3-released.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: news_post
+title: "Ruby 3.4.3 已发布"
+author: k0kubun
+translator: "GAO Jun"
+date: 2025-04-14 08:06:57 +0000
+lang: zh_cn
+---
+
+Ruby 3.4.3 已发布。
+
+此版本是包含了若干问题修正的定期更新版本。
+详细信息可参考 [GitHub 发布说明](https://github.com/ruby/ruby/releases/tag/v3_4_3) 。
```

Comment:
Extra space at the end of line.

Reply:
Extra space removed and a new version is pushed